client/js/team.ts

```diff
@@ -0,0 +1,52 @@
+const joinTeamButton = document.getElementById('joinTeam');
```

Comment:
Minor nitpick: could you make these single quotes consistent with the rest of the code by converting them to double quotes?